tasks/examTimetable.js

```diff
@@ -1,5 +1,78 @@
 'use strict';
 
+function parseExamPdf(fileData) {
+  const pdfjs = require('pdfjs-dist');
+  const moment = require('moment');
+  const _ = require('lodash');
+
+  const fileArray = new Uint8Array(fileData);
+  return pdfjs.getDocument(fileArray)
+    .then((pdfDocument) => {
```

Comment:
If we're gna use es6 for our parses then we better specify a node version in `.nvmrc`